### PR TITLE
PRESS4 304 | Store | Payments | simplified payment connection flow

### PR DESCRIPTION
**Pull request by sumathi3399**

### CODE REVIEW

File: src/components/ThirdPartyIntegration.js [hunk 1]
```diff
@@ -5,6 +5,14 @@ import { IntegrationsSdk } from "../sdk/integrations";
 import { PluginsSdk } from "../sdk/plugins";
 import { CaptiveRazorpay } from "./CaptiveRazorpay";
 import { Section } from "./Section";
+import { useLayoutEffect } from "@wordpress/element";
```
Comment (aratidgr8) [inline src/components/ThirdPartyIntegration.js:8]: @sumathi3399  Can we plz make use of `URLSearchParams` for changing query params values?

File: src/components/ThirdPartyIntegration.js [hunk 2]
```diff
@@ -22,21 +30,74 @@ export const ThirdPartyIntegration = ({
...
+
+    }
   });
+  const updateButtonEvent = async (event) => {
+    if (event.detail.connected == 1) {
+      const integrationStatusResponse =
+          await refreshIntegrationStatus();
+      if (integrationStatusResponse.complete) {
+        notify.push(`${id}-account-connect-success`, {
+          title: `Your ${id} account have been connected`,
```
Comment (aratidgr8) [inline src/components/ThirdPartyIntegration.js:81]: @sumathi3399 Can we plz add translations texts here?

File: src/components/Paypal.js [hunk 3]
```diff
@@ -58,6 +70,19 @@ const Paypal = ({ notify }) => {
                 >
                   {__("Manage", "wp-module-ecommerce")}
                 </Button>
+              ) : typeof yith_ppwc_login !== "undefined" &&
+                !parseInt(yith_ppwc_login?.liveConnected) ? (
+                <Link
+                  href={yith_ppwc_login.loginURL}
+                  variant="primary"
+                  style={{ color: "#fff" }}
```
Comment (aratidgr8) [inline src/components/Paypal.js:78]: @sumathi3399 Is it necessary to specify `color` inline? is it not included in `primary` styling

File: src/components/ThirdPartyIntegration.js (hunk 2 above)
Comment (sumathi3399) [reply src/components/ThirdPartyIntegration.js]: done

File: src/components/Paypal.js (hunk 3 above)
Comment (sumathi3399) [reply src/components/Paypal.js]: removed the inline styling and added the text color in className

**Review by aratidgr8 — APPROVED** [review]

**Pull request merged**